### ISSUE TITLE
com.squareup.picasso.Callback.onError(java.lang.Exception)" 

### ISSUE THREAD

**Issue by Mehran-Amp** [daimajia/AndroidImageSlider#416]
I was having this issue with an older version of Picasso 2.5.2. Solved it by upgrading to the newer version

implementation 'com.squareup.picasso:picasso:2.71828'
Don't forget that all calls to Picasso must now be updated from:

old syntax

Picasso.with(context)
    .load(url)
    .into(imageView)
to new syntax

Picasso.get()
    .load(url)
    .into(imageView)

please fix it

**Comment by Vengat121** [comment]
do you get any solution? @Mehran-Amp

**Comment by Mehran-Amp** [comment]
This is my App ( Build ) and worked good.

apply plugin: 'com.android.application'

android {
    compileSdkVersion 29
    defaultConfig {
        applicationId "com.example.banehersalcom"
        minSdkVersion 16
        targetSdkVersion 29
        versionCode 1
        versionName "1.0"
        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
        }
    }
}

dependencies {
    implementation fileTree(dir: 'libs', include: ['*.jar'])
    implementation 'androidx.appcompat:appcompat:1.0.2'
    implementation 'com.google.android.material:material:1.0.0'
    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
    testImplementation 'junit:junit:4.12'

    implementation 'com.github.JakeWharton:ViewPagerIndicator:2.4.1'
    implementation 'androidx.vectordrawable:vectordrawable:1.0.1'
    implementation 'com.github.JakeWharton:ViewPagerIndicator:2.4.1'
    implementation 'com.github.bumptech.glide:glide:4.9.0'
    annotationProcessor 'com.github.bumptech.glide:compiler:4.9.0'
    implementation 'androidx.recyclerview:recyclerview:1.0.0'
    implementation 'androidx.cardview:cardview:1.0.0'

    implementation 'com.squareup.picasso:picasso:2.5.2'

    implementation 'com.nineoldandroids:library:2.4.0'
    implementation 'com.daimajia.slider:library:1.1.5@aar'
    implementation 'com.borjabravo:readmoretextview:2.1.0'
    androidTestImplementation 'androidx.test:runner:1.2.0'
    androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
    implementation 'androidx.gridlayout:gridlayout:1.0.0'

}

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by Vengat121** [comment]
is there any solution without degrading picasso from 2.71828 to 2.5.2

**Comment by Mehran-Amp** [comment]
Yes merge the (daimajia/AndroidImageSlider)library with your project and
Do  this:


old syntax

Picasso.with(context)
.load(url)
.into(imageView)

to new syntax

Picasso.get()
.load(url)
.into(imageView)


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________
